warehouse/models/mart/gtfs/fct_hourly_rt_feed_files.sql

```diff
@@ -34,6 +34,23 @@ parse_outcomes AS (
     {% endif %}
 ),
 
+daily_totals AS (
+
+    SELECT
+
+        dt,
+        base64_url,
+        feed_type,
+        COUNT(*) as file_count_day
```

Comment:
Genuine question for @atvaccaro about style/best practices -- we already have something very similar [here](https://github.com/cal-itp/data-infra/blob/main/warehouse/models/mart/gtfs/fct_daily_rt_feed_files.sql#L35), you could reconstruct the daily total just by summing those two columns right (`parse_success_file_count` and `parse_failure_file_count`) -- that is more efficient, right?